src/modules/storage/LedgerStorage.ts

```diff
@@ -1221,7 +1221,7 @@ export class LedgerStorage extends Storages
         `SELECT utxos.address,
                 enrollments.enrolled_at,
                 enrollments.utxo_key as stake,
-                enrollments.random_seed,
+                enrollments.commitment,
```

Comment:
https://github.com/bosagora/stoa/blob/71702931ae267107f3517ccdf0c96d79498bc151/src/Stoa.ts#L224
https://github.com/bosagora/stoa/blob/71702931ae267107f3517ccdf0c96d79498bc151/src/Stoa.ts#L318
This is also affected.

Reply:
Updated.